swri_route_util/src/route_speeds.cpp

```diff
@@ -355,10 +355,24 @@ void speedsForObstacles(
       }
       continue;
     }
+    
+    // Use lane width as the car radius when smaller
```

Comment:
Please elaborate the comment to describe the purpose of this override and replace 'lane width' with the name of the propery